inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/service/RecommendationServiceImpl.java

```diff
@@ -166,6 +167,16 @@ public void deleteRecommender(Recommender aRecommender)
         return settings;
     }
     
+    @Override
+    @Transactional
+    public List<LoggedEvent> listLoggedEvents(Project aProject, User user)
```

Comment:
Let's put this method into the  `EventRepository` in the `inception-log` module. Also let's change it such that the type of event to be retrieved can be passed as a method argument.

I'd suggest to change the `User aUser` argument into a `String aUsername` argument because in that way we do not necessarily have to retrieve a `User` object before the method can be used.

Reply:
Done.